java/ql/src/experimental/CWE-532/SensitiveInfoLog.ql

```diff
@@ -0,0 +1,82 @@
+/**
+ * @id java/sensitiveinfo-in-logfile
+ * @name Insertion of sensitive information into log files
+ * @description Writing sensitive information to log files can give valuable guidance to an attacker or expose sensitive user information.
+ * @kind problem
+ * @tags security
+ *       external/cwe-532
+ */
+
+import java
+import semmle.code.java.dataflow.TaintTracking
+import DataFlow
+import PathGraph
```

Comment:
This import should only be included in `path-problem` queries.  Now this query could easily be turned into a `path-problem` query, but that's a separate concern.

Reply:
Hi @aschackmull ,

As I've revamped the query to use DataFlow::Configuration, which is pretty efficient now, and I'm not sure how path-problem query works, please take a look at the new query and provide more guidance if more changes are desired.

Thanks,
@luchua-bc

Reply:
Thanks for your help, I've converted the query to a path-problem query as mentioned below.